test/parallel/test-zlib-truncated.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+// tests zlib streams with truncated compressed input
+
+var common = require('../common');
+var assert = require('assert');
+var zlib = require ('zlib');
```

Comment:
Please use `const` for values that won't be changed that you either added or modified in this commit. :)